### ISSUE TITLE
Plugin not working with flutter 3.0

### ISSUE THREAD

**Issue by rajiphilip** [wilburt/flutter_paystack#120]
Hello, 

With the new flutter sdk 3.0 flutter_paystack is not working as it is throwing error.

**Comment by jobic10** [comment]
Update your `pubspec.yaml`
 ```
  # Temporarily using this until we can resolve the issue with the plugin.
  # Issue is: https://github.com/wilburt/flutter_paystack/issues/117
  flutter_paystack:
    git:
      url: https://github.com/definitelyme/flutter_paystack.git
      ref: a4a33c3dd0a12f46d655a2e63d11e9f20ba82d01

```

[Here](https://github.com/wilburt/flutter_paystack/issues/117#issuecomment-1128705003)

**Comment by Yoda-Man** [comment]
This is a big problem when is the package getting refreshed

**Comment by wilburt** [comment]
This has been fixed in 1.0.6. Can you update the package version?
